books/get_the_books.sh

```diff
@@ -0,0 +1,3 @@
+while read -r url; do
```

Comment:
This works!  Using google to find out the details is totally a way to go.  The hard part is knowing you can do something and having gone through it.  Your loop is actually interesting and different than some of the others... kudos for using `read`, all the others used `for`... nice